test/runtime/chaos.go

```diff
@@ -194,4 +195,34 @@ var _ = Describe("RuntimeChaos", func() {
 			check.res.ExpectSuccess("Curl from app2 to httpd1 should work but it failed at %s", check.time)
 		}
 	})
+
+	It("Validate that delete events on KVStore do not release in use identities", func() {
+		// This validates that if a kvstore delete event is send the identity
+		// is not release if it is in use. For more info issue #7240
+
+		prefix := "http://127.0.0.1:8500/v1/kv/cilium/state/identities/v1/id"
```

Comment:
Is there a way we can get this prefix in the kvstore from the package itself? Not sure if that'd require factoring certain stuff into a 'constants' package within the kvstore, though.